### PR TITLE
Add test for multiple models served with WatsonX stack in the same namespace

### PR DESCRIPTION
**Pull request by bdattoma**
This PR is adding an automated test case to check if it is possible to deploy and query multiple models in the same namespace, using KServe and Caikit runtime.

Moreover:
- it applies some fixes to run the automated tests against latest RHODS build we received
- it replaces caikit runtime image tag with "stable" to get always the latest changes

### CODE REVIEW

File: ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/422__model_serving_llm.robot [hunk 1]
```diff
@@ -58,12 +56,43 @@ Verify User Can Serve And Query A Model
...
     ...    namespace=${TEST_NS}
     ${host}=    Get KServe Inference Host Via CLI    isvc_name=${flan_isvc_name}   namespace=${TEST_NS}
-    ${body}=    Set Variable    '{"text": "At what temperature does liquid Nitrogen boil?"}'
+    ${body}=    Set Variable    '{"text": "At what temperature does water boil?"}'
     ${header}=    Set Variable    'mm-model-id: ${model_name}'
     Query Model With GRPCURL   host=${host}    port=443
     ...    endpoint="caikit.runtime.Nlp.NlpService/TextGenerationTaskPredict"
     ...    json_body=${body}    json_header=${header}
     ...    insecure=${TRUE}
+    [Teardown]    Run And Return Rc And Output    oc delete project ${TEST_NS}
```
Comment (bdattoma) [inline ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/422__model_serving_llm.robot:65]: it's a temp solution to facilitate local testing, will make it better

File: ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/422__model_serving_llm.robot [hunk 2]
```diff
@@ -58,12 +56,43 @@ Verify User Can Serve And Query A Model
...
+    Wait For Pods To Be Ready    label_selector=serving.kserve.io/inferenceservice=${model_two_isvc_name}
+    ...    namespace=${TEST_NS}
+    ${host}=    Get KServe Inference Host Via CLI    isvc_name=${flan_isvc_name}   namespace=${TEST_NS}
+    ${body}=    Set Variable    '{"text": "At what temperature does water boil?"}'
+    ${header}=    Set Variable    'mm-model-id: ${model_one_name}'
+    Query Model With GRPCURL   host=${host}    port=443
+    ...    endpoint="caikit.runtime.Nlp.NlpService/TextGenerationTaskPredict"
+    ...    json_body=${body}    json_header=${header}
+    ...    insecure=${TRUE}
+    [Teardown]    Run And Return Rc And Output    oc delete project ${TEST_NS}
```
Comment (bdattoma) [inline ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/422__model_serving_llm.robot:95]: it's a temp solution to facilitate local testing, will make it better

**Review by jgarciao — APPROVED** [review]

File: ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/422__model_serving_llm.robot [hunk 3]
```diff
@@ -46,24 +47,48 @@ Verify External Dependency Operators Can Be Deployed
...
-    ${host}=    Get KServe Inference Host Via CLI    isvc_name=${flan_isvc_name}   namespace=${TEST_NS}
-    ${body}=    Set Variable    '{"text": "At what temperature does liquid Nitrogen boil?"}'
-    ${header}=    Set Variable    'mm-model-id: ${model_name}'
+    ${host}=    Get KServe Inference Host Via CLI    isvc_name=${flan_model_name}   namespace=${TEST_NS}
+    ${body}=    Set Variable    '{"text": "${EXP_RESPONSES}[queries][0][query_text]"}'
+    ${header}=    Set Variable    'mm-model-id: ${flan_model_name}'
     Query Model With GRPCURL   host=${host}    port=443
     ...    endpoint="caikit.runtime.Nlp.NlpService/TextGenerationTaskPredict"
     ...    json_body=${body}    json_header=${header}
     ...    insecure=${TRUE}
```
Comment (lugi0) [inline ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/422__model_serving_llm.robot:65]: Shouldn't we check the response with `Model Response Should Match The Expectation` here?

Comment (bdattoma) [reply ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/422__model_serving_llm.robot:65]: yeah, that's a miss. However the same model is tested in the new test case. I will add the check in the next PR for consistency

**Pull request merged**